src/script/builder/tir/stmt.h

```diff
@@ -109,7 +109,7 @@ class AllocateConstFrameNode : public TIRFrameNode {
   DataType dtype;
   Array<PrimExpr> extents;
   tvm::runtime::NDArray data;
-  Buffer buffer{nullptr};
+  tvm::tir::Buffer buffer{nullptr};
```

Comment:
i guess we don't need `nullptr` here given Buffer is nullable

```suggestion
  tvm::tir::Buffer buffer;
```